hooks/pre-commit

```diff
@@ -12,6 +12,26 @@
 #
 # This script does not handle file names that contain spaces.
 
+govet=$(go vet ./lib)
+if [[ ! -z "$govet" ]]; then
+	echo "go vet checks not passed: Exit code $govet \n"
+	exit 1
+fi
+
+lint=$(golint -set_exit_status)
```

Comment:
Right here, you probably want to use `golint -set_exit_status some_files` to lint our staged files. At present this command will always exit with status code 0 since no files are passed to it. Also, I think you intended for the variable name `$lint` to be `$golint`, as it is referenced later on.